### ISSUE TITLE
Create the service

### ISSUE THREAD

**Issue by nobodyinperson** [nobodyinperson/co2monitor#4]
Create the background service with following abilities:
- [x] automatically started by **udev rule**
- [x] configurable via **configuration file**
- [x] logging of service messages to **logfile**
- [x] logging of measurements to **csv files**

**Comment by nobodyinperson** [comment]
The service provides now basic functionality:

- automatically started by **udev rule**
- configurable via **configuration file**
- logging of service messages to **logfile**
- logging of measurements to **csv files**

**Comment by nobodyinperson** [comment]
There is a new plan for the structure of the service. This aims towards **dbus** integration and easy handling of **multiple attached deviced** in parallel. The new structure plan is:
- [x] There will be an **invoker** that is started, when a device is plugged in
- [x] The **service** is started at bootup or on demand by the **invoker**
- [x] The service is actually just a **dbus** service that does the bridge between **logger threads** and the system, e.g. the **gui application**
- [x] The service starts the **logger threads** on **dbus** demand by the **invoker**

**Comment by nobodyinperson** [comment]
Done!
